apiClients/usersAPi.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const usersApi = axios.create({
+  baseURL: "/api/users",
+  headers: {
+    "Content-Type": "application/json",
+  },
+});
+
+export default usersApi;

```

Comment:
To w sumie jest zbędne, wystarczy jeden clientApi ktory bedzie mial cały ten poczatek, localhost etc i tak jak pisałem w poprzednim komentarzu, będziesz sobie koncówki dodawał, tj. users, usersTasks itd